datastore/src/main/java/io/spine/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -115,11 +115,12 @@ public int readEventCountAfterLastSnapshot(I id) {
         checkNotNull(id);
 
         RecordId datastoreId = toRecordId(id);
-        Optional<Int32Value> count = propertyStorage.read(datastoreId,
-                                                          Int32Value.getDescriptor());
-        int countValue = count.map(Int32Value::getValue)
-                              .orElse(0);
-        return countValue;
+        Optional<Message> optional = propertyStorage.read(datastoreId, Int32Value.getDescriptor());
+        if (!optional.isPresent()) {
+            return 0;
```

Comment:
I don't quite understand why did we need this change. As for me, the previous way was a bit more readable and much less ceremonial.

Reply:
The change was made to avoid suppression of unchecked casts.  The return type of `read()` changed from returning `Optional` of unchecked generic param `M` to `Optional<Message>`.